modules/opensimplex/noise_texture.cpp

```diff
@@ -202,12 +216,25 @@ void NoiseTexture::set_as_normalmap(bool p_as_normalmap) {
 	if (p_as_normalmap == as_normalmap) return;
 	as_normalmap = p_as_normalmap;
 	_queue_update();
+	_change_notify();
 }
 
 bool NoiseTexture::is_normalmap() {
 	return as_normalmap;
 }
 
+void NoiseTexture::set_bump_strength(float p_bump_strength) {
+
+	if (p_bump_strength == bump_strength) return;
+	bump_strength = p_bump_strength;
+	_queue_update();
```

Comment:
I would check for `as_normalmap` before queuing the update. The rest looks great :)

Reply:
Should be good now!